src/main/resources/lang/de_de.json

```diff
@@ -0,0 +1,121 @@
+	"popup_menu.calls": "Aufrufe anzeigen",
+	"popup_menu.calls.specific": "Aufrufe anzeigen (spezifisch)",
+	"popup_menu.declaration": "Zur Deklaration gehen",
+	"popup_menu.back": "Zurück gehen",
+	"popup_menu.forward": "Vorwärts gehen",
+	"popup_menu.mark_deobfuscated": "Als deobfuskiert markieren",
+	"popup_menu.reset_obfuscated": "Zu obfuskiert zurücksetzen",
+	"popup_menu.zoom.in": "Vergrößern",
+	"popup_menu.zoom.out": "Verkleinern",
+	"popup_menu.zoom.reset": "Größe zurücksetzen",
+
+	"info_panel.classes.obfuscated": "Nicht deobfuskierte Klassen:",
+	"info_panel.classes.deobfuscated": "Deobfuskierte Klassen:",
+	"info_panel.identifier": "Informationen zum Bezeichner:",
+	"info_panel.identifier.none": "Es wurde kein Bezeichner ausgewählt.",
+	"info_panel.identifier.variable": "Variable",
+	"info_panel.identifier.field": "Feld",
+	"info_panel.identifier.method": "Methode",
+	"info_panel.identifier.constructor": "Konstruktur",
+	"info_panel.identifier.class": "Klasse",
+	"info_panel.identifier.type_descriptor": "Typdeskriptor",
+	"info_panel.identifier.method_descriptor": "Methodendeskriptor",
+	"info_panel.identifier.modifier": "Zugriffsmodifikator",
+	"info_panel.identifier.index": "Index",
+	"info_panel.editor.class.decompiling": "(wird dekompiliert …)",
+	"info_panel.editor.class.not_found": "Klasse konnte nicht gefunden werden:",
+	"info_panel.tree.inheritance": "Vererbung",
+	"info_panel.tree.implementations": "Implementierungen",
+	"info_panel.tree.calls": "Aufrufe",
+
+	"progress.operation": "%s – Vorgang läuft",
+	"progress.jar.indexing": "Java‐Archiv wird indiziert:",
+	"progress.jar.indexing.entries": "Einträge …",
+	"progress.jar.indexing.references": "Referenzen der Einträge …",
+	"progress.jar.indexing.methods": "Brückenmethoden …",
+	"progress.jar.indexing.process": "Verarbeiten …",
+	"progress.jar.writing": "Java‐Archiv wird geschrieben …",
+	"progress.sources.writing": "Quelldateien werden geschrieben …",
+	"progress.classes.deobfuscating": "Klassen werden deobfuskiert …",
+	"progress.classes.decompiling": "Klassen werden dekompiliert …",
+	"progress.mappings.enigma_file.loading": "Zuordnungsdatei wird geladen",
+	"progress.mappings.enigma_file.done": "Fertig!",
+	"progress.mappings.enigma_file.writing": "Klassen werden geschrieben …",
+	"progress.mappings.enigma_directory.loading": "Zuordnungsdatei wird geladen",
+	"progress.mappings.enigma_directory.writing": "Klassen werden geschrieben",
+	"progress.mappings.tiny_file.loading": "Zuordnungsdatei wird geladen",
+	"progress.mappings.tiny_v2.loading": "Zuordnungsdatei wird geladen",
+	"progress.mappings.srg_file.generating": "Zuordnungen werden generiert",
+	"progress.mappings.srg_file.writing": "Zuordnungen werden geschrieben",
+	"progress.stats": "Statistiken werden generiert",
+	"progress.stats.data": "Daten werden generiert",
+
+	"javadocs.edit": "Javadoc-Kommentare bearbeiten",
+	"javadocs.instruction": "Bearbeiten Sie hier die Javadoc-Kommentare.",
```

Comment:
Simply use "Javadoc" here for `javadocs.edit` and `javadocs.instruction`.
-> "Javadoc bearbeiten"
-> "Bearbeiten Sie hier die Javadoc"

c. f. https://github.com/FabricMC/Enigma/pull/227/files#r410940749